src/Pages/Shop/ShopListCol.js

```diff
@@ -1,7 +1,55 @@
-import React, { Component } from "react";
+import React from "react";
+import GoodsList from "../../Components/GoodsList";
+import "./ShopListCol.scss";
+
+class ShopListCol extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      mouseover: true,
+      Goods: [],
+    };
+  }
+
+  componentDidMount = () => {
+    fetch("Data/MockData/Goods.json")
+      .then((response) => response.json())
+      .then((response) => {
+        this.setState({ Goods: response.Goods });
+      });
```

Comment:
```suggestion
      .then(({ Goods }) => this.setState({ Goods }));
```
- 함수 파라미터에 직접 구조분해 할당 해줄 수 있습니다
- arrow function을 축약형으로 쓰면 한줄로 표현할 수 있습니다
- 객체 단축 속성명을 사용하면 객체의 키값과 value 자리에 들어오는 변수명이 같을 경우 하나로 합쳐줄 수 있습니다

Reply:
오

Reply:
수정완료하였습니다